recipes/rdock/meta.yaml

```diff
@@ -1,47 +1,60 @@
-package:
-  name: rdock
-  version: 2013.1
+{% set name = "rDock" %}
+{% set version = "0.22" %}
 
-build:
-  number: 2
-  skip: True  # [osx]
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: https://depot.galaxyproject.org/software/rDock/rDock_2013.1_src_all.tar.gz
-  sha256: e716998c3f8a2a70205a8d30ba22675bfdb1764d13c858645138c5eadf2a37e9
+  url: https://github.com/CBDD/rDock/releases/download/v{{ version }}/{{ name }}_2022_v0.tar.gz
+  sha256: 3dd4d4eae8cdbcfd5cec8584df43f5fc4fba2267b6c7041bf93e96705277ba62
   patches:
-    - rDock_python_fixes.patch
-    - rDock_fix_paths.patch
+    - 0001-Add-missing-cmath-for-cos-.-usage.patch
+
+build:
+  number: 0
+  skip: True  # [osx]
 
 requirements:
   build:
-    - make
+    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make
+    - perl *
   host:
     - cppunit
-    - perl
     - popt
-    - python
   run:
     - numpy
-    - perl
-    - popt
+    - perl *
+    - python *
     - openbabel
 
 test:
   commands:
+    - grep() { sed "/$*/{:m;n;bm};\$q1" ; }
```

Comment:
This is a cryptic hack to output the piped contents but still fail if the requested pattern is not found. That way we don't swallow the error message when we `2>&1 | grep`.
You can remove it or keep it ;).
(It works like this: Print every line as given by the input (no `s/.../.../` given). Return exit code 1 if the end of the file is reached (`$q1`). If there is a line with the pattern (`/$*/`), then go into a tight loop that just appends every line (set label `:m`, append the next one `n`, jump `b` back to `m`) -- when it reaches the end, the appended lines are printed and it exits normally.)

Reply:
Sure, sounds good (I have added a comment though 🤪)